.env

```diff
@@ -92,35 +92,65 @@ BALANCE_CREDIT=300000000000
 BALANCE_LOCKED=300000000000
 
 # ====================================== ORION ======================================
-# Orion db config
-# Overrides `DB_*` environment variables for orion services in docker-compose.yml
-# We use a single postgres service for both:
-# - Subsquid archive's database (squid-archive, squid-archive-gateway) and
-# - Orion's database (orion_processor, orion_graphql-server).
-# ORION_DB_NAME is the name of the Orion's database, while SQUID_ARCHIVE_DB_NAME
-# is the name of the Subsquid archive database.
-SQUID_ARCHIVE_DB_NAME=squid_archive
-ORION_DB_NAME=orion
-ORION_DB_USER=postgres
-ORION_DB_PASS=postgres
+# LOCAL DEV ENVIRONMENT
+
+ORION_ENV=development
+DEV_DISABLE_SAME_SITE=true
+
+# Db config
+ORION_DB_NAME=squid
+ORION_DB_PASS=squid
+DB_ADMIN_USER=admin
+DB_ADMIN_PASS=admin
 ORION_DB_PORT=5433
 
-# Orion / Subsquid archive service ports
-ORION_API_PORT=4350
+# Processor service prometheus port
 PROCESSOR_PROMETHEUS_PORT=3337
-SQUID_ARCHIVE_GATEWAY_PORT=8000
+# Graphql server port
+GRAPHQL_API_PORT=4350
+# Auth api port
+AUTH_API_PORT=4074
 
-# Other Orion configs
+APP_NAME=Gleev
 SUPPORT_NO_CATEGORY_VIDEOS=true
 SUPPORT_NEW_CATEGORIES=true
 KILL_SWITCH_ON=false
-VIDEO_VIEW_PER_IP_TIME_LIMIT=30
-VIDEO_RELEVANCE_VIEWS_TICK=50
-RELEVANCE_WEIGHTS="[1, 0.03, 0.3, 0.5]"
+# 10 seconds
+VIDEO_VIEW_PER_USER_TIME_LIMIT=10
+# Operator API secret
 OPERATOR_SECRET=this-is-not-so-secret-change-it
+# every 50 views video relevance score will be recalculated
+VIDEO_RELEVANCE_VIEWS_TICK=50
+# [
+#   newness (negative number of days since created) weight,
+#   views weight,
+#   comments weight,
+#   rections weights,
+#   [joystream creation weight, YT creation weight]
+# ]
+RELEVANCE_WEIGHTS="[1, 0.03, 0.3, 0.5, [7,3]]"
 MAX_CACHED_ENTITIES=1000
 APP_PRIVATE_KEY=this-is-not-so-secret-change-it
-TRUSTED_REVERSE_PROXIES=1
+SESSION_EXPIRY_AFTER_INACTIVITY_MINUTES=60
+SESSION_MAX_DURATION_HOURS=720
+EMAIL_CONFIRMATION_ROUTE=http://localhost:4074/api/v1/confirm-email?token={token}
```

Comment:
obviously this will only work when opening email from the machine. But I gather we would most likely disable email confirmation and notifications in a playground setup so thats fine.